devel/Dockerfile

```diff
@@ -109,6 +109,12 @@ RUN git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project.g
     cd .. && rm -rf phoenix-rtos-project && \
     chmod -R a+rX /opt/phoenix/utils
 
+# install tini (init system) for host, necessary to kill zombie processes used for example in exit() test during CI
```

Comment:
nit: you're not killing zombies but reaping them - zombie is an already killed process, you're just reading their exit information